minimum-constraints.txt

```diff
@@ -38,7 +38,8 @@ wheel==0.38.1; python_version >= '3.7'
 
 # Direct dependencies for runtime (must be consistent with requirements.txt)
 
-zhmcclient==1.12.1
```

Comment:
shouldn't you keep that line then?

Reply:
No, because in the requirements.txt file there is the git+http based link for zhmcclient and that already defines an exact version (the link cannot be stated such that it ends up being a minimum), so that no additional minimum version is needed here.

Also, the git+http link syntax (which we would need to use in order to be consistent with the requirements.txt file) is not allowed in a constraints file.